draft-ietf-tls-exported-authenticator.md

```diff
@@ -156,28 +167,34 @@ sender:
   finished key" for authenticators sent by the client and server respectively.
 
 The context_value used for the exporter is empty (zero length) for all four
-values. The length of the exported value is equal to the length of the output of
-the hash function selected in TLS for the pseudorandom function (PRF).  Cipher
-suites that do not use the TLS PRF MUST define a hash function that can be used
-for this purpose or they cannot be used.  For TLS 1.3 symmetric cipher suites,
-the hash algorithm used with HKDF is used.
-
-If the connection is TLS 1.2, the master secret MUST have been computed
-with the extended master secret {{!RFC7627}} to avoid key synchronization attacks.
+values.  There is no need to include additional context
+information at this stage since the application-supplied context
+is included in the authenticator itself.  The length of the exported value is equal to the length of the output of
+the hash function selected in TLS for the pseudorandom function (PRF).  Exported authenticators cannot be
+used with cipher suites that do not use the TLS PRF and have not defined
+a hash function for this purpose.  This hash is referred to as the authenticator hash.
+
+Let's rephrase this as "Exported Authenticators MUST NOT be generated or
+accepted on connectons not using the extended master secret {{!RFC7627}}
+extension, to avoid key synchronization attacks", to be very explicit
+about the prohibited behavior.
```

Comment:
This paragraph appears to be feedback containing a quoted suggestion, not the resulting text.

Reply:
Removing the framing might do, but watch for the typo on "connections"

Reply:
Indeed.